src/ansys/mapdl/core/mapdl_console.py

```diff
@@ -315,7 +316,7 @@ def kill(self):
                     self._log.warning("Unable to kill process %d", self._process.pid)
                 self._log.debug("Killed process %d", self._process.pid)
 
-    @property
+    @MapdlBase.name.getter
```

Comment:
Any reason for this change? I guess it is just more explicit. Any other reason?

Reply:
My type checker was complaining when I used `@property` in mapdl_inprocess, it expect to also have the setter:
```
"name" incorrectly overrides property of same name in class "_MapdlCore"
Property method "fset" is missing in override
```
I followed what was said in this response
https://stackoverflow.com/a/7020531

Reply:
Ohh. Ok ok. Good to know :)

Reply:
The problem is that some backend (mapdl_grpc and mapdl_console) use a function to define the name lazyly
One other solution would be to keep the `_name` only in MapdlCore but override the getter in those backends

Reply:
Dont worry if we need to change `MapdlGrpc` and/or `MapdlConsole`, if it is for good. I am ok with any option.